balboa-kafka/src/main/resources/config/config.properties

```diff
@@ -0,0 +1,47 @@
+# Logging.
+log4j.logger.com.socrata.balboa = DEBUG, console
+
+# Console configuration (used in development)
+log4j.appender.console = org.apache.log4j.ConsoleAppender
+log4j.appender.console.layout = org.apache.log4j.PatternLayout
+log4j.appender.console.layout.ConversionPattern =%p [%t] %d %c %m%n
+
+# The levels at which to summarize. After you start summarizing, you should not
+# ever change these existing. If you do, some tiers of summarization will contain
+# inconsistent data with the rest. 
+balboa.summaries = fifteen_minute,hourly,daily,monthly,yearly
+
+# Period of serialization to use. After you start summarizing, this should never
+# change or things will break really bad. Some datastores (e.g. SQL) use their
+# own serialization and ignore this value.
+balboa.serializer = protobuf
+
+# Data store to use.
+balboa.datastore = buffered-cassandra
+
+# Cassandra servers. Servers should be separated by a comma.
+cassandra.servers = localhost:9160
+cassandra.keyspace = Metrics2012
+
+cassandra.sotimeout = 120000
+cassandra.maxpoolsize = 10
+failfast.initialbackoff = 10000
+failfast.maxbackoff = 120000
+
+# Default duration to buffer incoming metrics in memory
+# balboa.datastore should be set to buffered-cassandra
+buffer.granularity = 120000
+
+
+# Kafka Properties
+####################
+
+zookeeper.connect = localhost:2181
+zookeeper.session.timeout.ms = 1000000
+group.id = test_group
+
+# Balboa defined Kafka Properties
+# The topic this specific service will be listening for
+balboa.kafka.topic = test_topic
+# The number of partitions
+balboa.kafka.topic.partitions = 8
```

Comment:
We set a default partition value of 8 here in the config file; but when we get the property we default it to 1. Why?